### ISSUE TITLE
Empty text strings being created and drawn

### ISSUE THREAD

**Issue by dmvaldman** [tangrams/tangram#447]
Somewhere in our pipeline we are creating an empty text string, likely as a default when no `name` field is given in some data source, and drawing it to the screen at tile corners. You can only notice this if collision boxes are drawn. Which looks like 

<img width="538" alt="screen shot 2016-11-11 at 11 30 10 am" src="https://cloud.githubusercontent.com/assets/796394/20222907/20165ca0-a805-11e6-9a0c-cc3d32b12fa3.png">

<img width="1395" alt="screen shot 2016-11-11 at 11 28 59 am" src="https://cloud.githubusercontent.com/assets/796394/20222915/24b61b56-a805-11e6-8c31-452b8fe2689b.png">

**Comment by bcamper** [comment]
Fixed in https://github.com/tangrams/tangram/commit/0b384bb4236a351338e146ee3a9f274cc8a1d399

These were being caused empty text strings for labels, e.g. values of `''`, as opposed to `null`/`undefined` values, which were already skipped.

In this case, it was caused by ocean features with an empty string `name` value. Any such empty string values should probably be dropped from our tiles (though Tangram should properly handle them nonetheless) -- I have opened an issue in tilezen for those features.
